dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/legal/LicenseRepositoryImpl.java

```diff
@@ -159,6 +160,18 @@ public String getColumnName(String modelProperty) {
     }
   }
 
+  @Override
+  public List<Locale> getLanguages() {
+    String query =
+        "SELECT DISTINCT jsonb_object_keys("
+            + tableAlias
+            + ".label) as languages FROM "
+            + tableName
+            + " AS "
+            + tableAlias;
+    return dbi.withHandle(h -> h.createQuery(query).mapTo(Locale.class).list());
+  }
```

Comment:
Könnte man das nicht mal in die Basisklasse verschieben? Dann wäre es gleich für alle `Identifiable`s verfügbar.

Reply:
`License` erbt nicht von `Identifiable` :wink: